openjdk8/src/main/docker/Dockerfile

```diff
@@ -25,18 +25,35 @@ RUN \
  && apt-get -y -q --no-install-recommends install \
     # install the jdk and its dependencies
     ca-certificates-java \
-    openjdk-${openjdk.version.major}-jdk-headless=${openjdk.version}'*' \
     # procps is used in the jvm shutdown hook
     procps \
     # other system utilities
     netbase \
     wget \
     unzip \
+    fontconfig \
+    locales \
 
  # cleanup package manager caches
  && apt-get clean \
  && rm /var/lib/apt/lists/*_*
 
+ARG OPENJDK_VER='jdk8u345-b01'
+ARG OPENJDK_VER_NAME='8u345b01'
+ARG OPENJDK_TEMP_PATH='/tmp/openjdk'
+ARG OPENJDK_OUT='/usr/lib/openjdk'
+ENV PATH $OPENJDK_OUT/bin:$PATH
+ENV JAVA_HOME $OPENJDK_OUT
+
+RUN mkdir -p $OPENJDK_TEMP_PATH \
+ && mkdir -p $OPENJDK_OUT \
```

Comment:
Any preferences on && over another `RUN` here? With one `RUN` for each command, the error message will more explicit when it fails. There might be performance impact, but I wouldn't imagine it's going to be significant enough.

Reply:
I picked && to follow the existing patterns we use in the Dockerfile. I also remember Ludo mentioned that single RUN creates single layer.

Reply:
combining RUN is better for minimizing layers.